DirectEmployers/MyJobs
# [PD-1757] outreach inbox management

## Goal

* Create back end handling for management of non-user outreach inboxes.
* Develop basic front end for modification
 * Can be revised/updated later, or used as-is

## To Test

* Check out branch
* Log in to a user with PRM access
* Head to /prm/view/nonuseroutreach/inboxes


## Notice

The format of the front end code is a bit off. There is a tag with "@my.jobs" at the end of the input textbox. As is, the model expects a full email to be entered into that textbox. This is being addressed in a following ticket. Until then, testing should be done with the current model design in mind.
